x-pack/plugins/fleet/common/experimental_features.ts

```diff
@@ -26,6 +26,7 @@ export const allowedExperimentalValues = Object.freeze<Record<string, boolean>>(
   outputSecretsStorage: true,
   remoteESOutput: true,
   agentless: false,
+  disableKQLValidation: true,
```

Comment:
I wonder if it's confusing to have a `disable` feature flag here. Our other flags use `enabled: true` as a pattern, e.g. `agentTamperProtectionEnabled: true`. What do you think about switching this around to `enableStrictKQLValidation: false` instead?

Reply:
I know what you mean, but this feature was originally enabled and now we are disabling it. I can still change it if you think it's too confusing though.